multik-api/build.gradle.kts

```diff
@@ -3,22 +3,89 @@
  */
 
 plugins {
+    kotlin("multiplatform")
+    id("maven-publish")
     val dokka_version: String by System.getProperties()
 
     id("org.jetbrains.dokka") version dokka_version
 }
 
-kotlin {
-    explicitApi()
+repositories {
+    mavenCentral()
 }
 
 val common_csv_version: String by project
-dependencies {
-    implementation(kotlin("reflect"))
-    implementation("org.apache.commons:commons-csv:$common_csv_version")
+
+kotlin {
+    explicitApi()
+    jvm {
+        compilations.all {
+            kotlinOptions.jvmTarget = "1.8"
+        }
+        testRuns["test"].executionTask.configure {
+            useJUnit()
+        }
+    }
+    mingwX64()
+    linuxX64()
+    macosX64()
+    macosArm64()
+    iosArm64()
+    iosSimulatorArm64()
+    iosX64()
+    js {
+        val timeoutMs = "1000000"
+        browser{
+            testTask {
+                useMocha {
+                    timeout = timeoutMs
```

Comment:
Honestly no idea. Never really used kotlin JS for anything, I just added JS support because someone asked. I can look. Also, I'm not sure if there's any point to having both browser and nodejs JS targets. I have to look into what the difference is.